### ISSUE TITLE
Switzerland: Replace unofficial feed "f-u0-switzerland" with official feed

### ISSUE THREAD

**Issue by stevevance** [transitland/transitland#296]
We are currently using an unofficial feed for national transit data in Switzerland, using the onestopID `f-u0-switzerland` and [this URL](http://gtfs.geops.ch/dl/gtfs_complete.zip).

It would be great if I could simply update `f-u0-switzerland` with the [new URL](https://opentransportdata.swiss/en/dataset/timetable-2017-gtfs/permalink), but there may be some dropping and adding of operators that I'll have to double check before hand. 

## Basic info
- Operators in current, unofficial feed: 420 
- Operators in official feed: 421

**Comment by irees** [comment]
I agree with the plan to update the existing url and update the agencies as necessary.

**Comment by stevevance** [comment]
I've replaced the URL today...waiting to enqueue it for import.

**Comment by stevevance** [comment]
The import failed multiple times because of...I'm not sure, [it looks like a timeout](https://transit.land/dispatcher/feed-versions/ed6c41349df9afe7ae97426449c64f9b1265a597). 

Also, I have a [changeset waiting to be applied](https://transit.land/dispatcher/changesets/9956) that would apply a new URL for 2018. 

@irees I've enqueued it again, just now.

**Comment by denysvitali** [comment]
I'm not sure if it's related, but apparently the `schedule_stop_pairs` doesn't return anything with any swiss operator (SBB, PostAuto, ...)  
  
Example:
http://transit.land/api/v1/schedule_stop_pairs?operator_onestop_id=o-u0-sbbschweizerischebundesbahnensbb

**Comment by denysvitali** [comment]
http://transit.land/api/v1/schedule_stop_pairs?feed_onestop_id=f-u0-switzerland doesn't return anything. I guess something's wrong on TransitLand end or the changeset wasn't applied succesfully, resulting in missing data
